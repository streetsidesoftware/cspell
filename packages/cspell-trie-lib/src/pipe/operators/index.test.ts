import * as operators from '.';

describe('Operators', () => {
    test('operators', () => {
        expect(Object.keys(operators).sort()).toMatchSnapshot();
        expect(operators.asyncAwait).toBeInstanceOf(Function);
        expect(operators.asyncFilter).toBeInstanceOf(Function);
        expect(operators.asyncFlatten).toBeInstanceOf(Function);
        expect(operators.asyncMap).toBeInstanceOf(Function);
        expect(operators.asyncUnique).toBeInstanceOf(Function);
        expect(operators.syncFilter).toBeInstanceOf(Function);
        expect(operators.syncFlatten).toBeInstanceOf(Function);
        expect(operators.syncMap).toBeInstanceOf(Function);
        expect(operators.syncUnique).toBeInstanceOf(Function);
        expect(operators.filter).toBeInstanceOf(Function);
        expect(operators.flatten).toBeInstanceOf(Function);
        expect(operators.map).toBeInstanceOf(Function);
        expect(operators.unique).toBeInstanceOf(Function);
    });
});
