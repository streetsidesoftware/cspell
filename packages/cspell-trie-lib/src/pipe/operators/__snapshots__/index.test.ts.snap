// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Operators operators 1`] = `
Array [
  "opAwaitAsync",
  "opFilter",
  "opFilterAsync",
  "opFilterSync",
  "opFlatten",
  "opFlattenAsync",
  "opFlattenSync",
  "opJoinStrings",
  "opJoinStringsAsync",
  "opJoinStringsSync",
  "opMap",
  "opMapAsync",
  "opMapSync",
  "opUnique",
  "opUniqueAsync",
  "opUniqueSync",
]
`;
